### PR TITLE
CI: add job names, add Node.js v14 to matrix

### PR DESCRIPTION
**Pull request by boneskull**
also fix webhooks

Ref: #4198

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -75,19 +85,23 @@ jobs:
...
-  urls:
-    # for gitter mochajs/mocha
-    - secure: fUrHenYJs+pTuLtgBRoYyrlyfVekxaIGmLWq7bhUUqBj/7p5eCkQFn13LlPht0/4WWZOiPBcdTN7tKnz3Ho7ATUJhAchvOWDUgL5gtTvOzeCHbPuCvHz/VLK6hMoPdbLA45M864NDLotfHvyh62WgQaVw9iPc80eb+umaDPrYiU=
-    # for gitter mochajs/contributors
-    - secure: rGMGYWBaZgEa9i997jJHKzjI8WxECqLi6BqsMhvstDq9EeTeXkZFVfz4r6G3Xugsk3tFwb/pDpiYo1OK36kA5arUJTCia51u4Wn+c7lHKcpef/vXztoyucvw6/jXdVm/FQz1jztYYbqdyAOWC2BV8gYvg5F8TpK05UGCe5R0bRA=
-  on_success: change
-  on_failure: always
+  webhooks:
+    urls:
+      # for gitter mochajs/mocha
```
Comment (juergba) [inline .travis.yml:102]: IMO we don't need this for mochajs/mocha, it doesn't seem to work anyway in this room.
In mochajs/contributors it works.

Comment (boneskull) [reply .travis.yml:102]: k

Comment (boneskull) [reply .travis.yml]: removed the main gitter room webhook

**Comment by boneskull** [comment]
> What about Node v14 in appveyor?

I'll add it.

**Comment by boneskull** [comment]
> The tests in Node v14 run very slow.

Where do you see this?

**Pull request merged**